src/test/videopress/edit.js

```diff
@@ -37,8 +36,17 @@ import {
 	pressSettingInPanel,
 	pressSettingInPicker,
 	generateFetchMocks,
+	generateBlockHTML,
 } from './local-helpers/utils';
 
+const VIDEO_TITLE = 'default-title-is-file-name';
+const FETCH_MOCKS_METADATA = {
+	title: VIDEO_TITLE,
+	description: '',
+	post_id: 1,
+	duration: 2803,
```

Comment:
To be honest, these parameters don't provide any value in the snapshot. They aren't meant to be updated in the test cases but since the VideoPress block updates settings automatically from the API when rendering the block, they are included. I'm happy with both options, but if including them in the tests gives a wrong impression of their usage, I can revert it.

Reply:
I have no strong preference. I suppose I mostly wanted to convey that we should not be afraid of updating snapshots. If a given snapshot is too unwieldily to review, a more appropriate alternative may be forgoing snapshots altogether and asserting against specific outcomes, e.g. attributes within the snapshot.

Reply:
> I have no strong preference. I suppose I mostly wanted to convey that we should not be afraid of updating snapshots.
Yep, I think in this case we can update the snapshots as we don't have that many to make the review complex 👍.

> If a given snapshot is too unwieldily to review, a more appropriate alternative may be forgoing snapshots altogether and asserting against specific outcomes, e.g. attributes within the snapshot.

Good point. In general, I'd rather lean on snapshots to simplify test logic. But I agree that in some cases there's a good value on asserting specific outcomes. For these test cases, I'm hesitant about which one would fit better. For now, I'd prefer to keep them as-is and only focus on the fixes, if that makes sense.

Reply:
> For now, I'd prefer to keep them as-is and only focus on the fixes, if that makes sense.

Absolutely. 🚀

Reply:
I've just applied this suggestion via https://github.com/wordpress-mobile/gutenberg-mobile/pull/5916/commits/68a1ccd7f4ab9334b596e042606887bed4162813.